lib/lograge/sql.rb

```diff
@@ -15,13 +15,16 @@ class << self
       attr_accessor :min_duration_ms
       # Filter SQL query
       attr_accessor :query_filter
+      # Filter wich SQL queries to store
+      attr_accessor :query_name_denylist
 
       # Initialise configuration with fallback to default values
       def setup(config)
         Lograge::Sql.formatter       = config.formatter       || default_formatter
         Lograge::Sql.extract_event   = config.extract_event   || default_extract_event
         Lograge::Sql.min_duration_ms = config.min_duration_ms || 0
         Lograge::Sql.query_filter    = config.query_filter
+        Lograge::Sql.query_name_denylist = config.query_name_denylist || [/\ASCHEMA\z/, /\ASolidCable::/]
```

Comment:
@fdaugs @iMacTia 
Perhaps this could be better for performance and avoiding runtime error.

```suggestion
        Lograge::Sql.query_name_denylist = Regexp.union(config.query_name_denylist || [/\ASCHEMA\z/, /\ASolidCable::/])
```

related: https://github.com/rails/rails/pull/46452

Reply:
@kakubin Thanks for the suggestion! https://github.com/iMacTia/lograge-sql/pull/63